scripts/e2e/test-access-check.sh

```diff
@@ -0,0 +1,44 @@
+set -x
```

Comment:
Are you gonna integrate those to the existing e2e framework?

Reply:
yes, will modify this now.